ui/src/Explore/Components/ContentView.tsx

```diff
@@ -104,15 +109,34 @@ const ContentView = (props: { tableContent: Content }): React.ReactElement => {
     );
   };
 
-  const renderView = (sqlViewContent: SqlView) => {
-    const { sqlText, dialect } = sqlViewContent;
+  const renderIcebergView = (icebergViewContent: IcebergView) => {
+    const { metadataLocation, versionId, schemaId, sqlText, dialect } =
+      icebergViewContent;
     return (
       <div className={"ContentView"}>
         <div className={"ContentView__contentWrapper"}>
           <div>
             <span>Type: </span>
             <span className={"ml-2"}> {tableContent.type}</span>
           </div>
+          {metadataLocation && (
+            <div>
+              <span>Metadata Location:</span>
+              <span className={"ml-2"}>{metadataLocation}</span>
+            </div>
+          )}
+          {versionId && (
```

Comment:
We don't show "details" for `IcebergTable` (above). Either for both of for none ;)